draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -388,7 +385,7 @@ data. The original JSON data is then used by the application. See
 ## Format of an SD-JWT
 
 An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The
-payload of an SD-JWT MUST contain the `sd_digest_derivation_alg` claim
+payload of an SD-JWT MUST contain the `sd_hash_alg` claim
```

Comment:
I'd like to be consistent with the sd names.  So either 

`_sd` and `_sd_hash_alg` 

or

`sd` and `sd_hash_alg`

But not mixed.

Reply:
+1 for `_sd_hash_alg`. 

BTW, I added the underscore in front of `_sd` because it sorts nicely to the top (in case the claims are sorted alphabetically by key). With just `sd` the hashes sit somewhere in between the other non-sd claims.

Reply:
`_sd_alg` may work as well.

Reply:
`_sd_alg` would work for me

Reply:
I think `_sd_hash_alg` is explicit and nice?

Reply:
I'd prefer `_sd_alg`. Size isn't everything but keeping JWTs as small as reasonable is worth keeping in mind. And JWT claim names are typically rather short/concise. `_sd_alg` is shorter but still conveys enough about what it is.